dipy/workflows/tests/test_reconst_dti.py

```diff
@@ -23,7 +23,14 @@ def test_reconst_dti_alt_tensor():
                       extra_kwargs={'nifti_tensor': False})
 
 
-def reconst_flow_core(flow, extra_args=[], extra_kwargs={}):
+def reconst_flow_core(flow, extra_args=None, extra_kwargs=None):
+
+    if extra_args is None:
+        extra_args = []
+
+    if extra_kwargs is None:
+        extra_kwargs = {}
```

Comment:
`func_args = func_args or []` instead

`func_kwargs = func_kwargs or {}` instead

Reply:
Done in f49517e.